GitUI/CommandsDialogs/RevisionFileTree.cs

```diff
@@ -493,14 +494,13 @@ private void FileTreeContextMenu_Opening(object sender, System.ComponentModel.Ca
             }
 
             saveAsToolStripMenuItem.Visible = isFile;
-            resetToThisRevisionToolStripMenuItem.Visible = isFile && !Module.IsBareRepository();
-            toolStripSeparatorFileSystemActions.Visible = isFile;
+            resetToThisRevisionToolStripMenuItem.Visible = isFileOrFolder && !Module.IsBareRepository();
+            toolStripSeparatorFileSystemActions.Visible = isFileOrFolder;
 
-            fileHistoryToolStripMenuItem.Enabled = itemSelected;
```

Comment:
I don't understand the meaning of this change... Could you explain?

Reply:
Can revert. The check is not really needed, the other checks are to avoid exceptions when the menu is opening without selections (which could happen, not easy to reproduce).